src/lib/falcor/FalcorController.js

```diff
@@ -34,41 +34,76 @@ export default class FalcorController extends BaseComponent {
   // this should always contain all the data the component needs
   loadFalcorCache() {
     console.log("LOADING FROM CACHE")
-    const falcorPath = this.constructor.getFalcorPath(this.props.params)
-
-    const data = this.props.model.getCache(falcorPath)
+    let falcorPath = this.constructor.getFalcorPath(this.props.params)
```

Comment:
Should be called falcorPathSets to indicate it is a list of several falcorPathSet